CONTRIBUTING.md

```diff
@@ -37,6 +37,10 @@ This section should get you running with **Amplify CLI** and get you familiar wi
 
    > If you are using Yarn v2, run `yarn set version classic` to change to Yarn Classic.
 
+   > Ensure that [Yarn global bin](https://classic.yarnpkg.com/en/docs/cli/global) is added to your PATH. For example, add `export PATH="$(yarn global bin):$PATH"` to your shell profile file on Linux or macOS.
+
+1. Ensure you have [Java](https://adoptopenjdk.net/) installed and `java` command is available in your system. This is required for DynamoDB emulator.
```

Comment:
Small nit, can you bump the numbers that follow by 1?

Reply:
Also, that link looks out of date. I think the correct link would be https://adoptium.net/

Reply:
They're all `1.` see the [raw source](https://raw.githubusercontent.com/aws-amplify/amplify-cli/dev/CONTRIBUTING.md). Markdown auto numbers the points.

Reply:
Link fixed. Thanks for catching this.